packages/bpk-component-infinite-scroll/index.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import withInfiniteScroll from './src/withInfiniteScroll';
+
+export { default as onItemsFetch } from './src/onItemsFetch';
+
+export default withInfiniteScroll;
```

Comment:
we need to export the prop types here too, see: https://github.com/Skyscanner/backpack/blob/master/packages/bpk-component-navigation-bar/index.js#L32